openapi/eloqua/Ballerina.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+distribution = "slbeta6"
+license = ["Apache-2.0"]
+keywords = ["Marketing/Marketing Automation", "Cost/Paid"]
+org = "ballerinax"
+name = "eloqua"
+icon = "icon.png"
+repository = "https://github.com/ballerina-platform/ballerinax-openapi-connectors"
+version = "1.0.1"
```

Comment:
What is the version we need to give for newly added connectors ?

Reply:
Yes. Usually we add `1.0.0` for the new connectors.

Reply:
Updated to `1.0.0`